fastlane/lib/fastlane/actions/create_app_online.rb

```diff
@@ -30,7 +30,7 @@ def details
           'Create new apps on iTunes Connect and Apple Developer Portal via _produce_.',
           'If the app already exists, `create_app_online` will not do anything.',
           'For more information about produce, visit its GitHub page:',
```

Comment:
This is now not true any more. Replace with something more appropriate, like "documentation".